integrated_channels/integrated_channel/course_metadata.py

```diff
@@ -0,0 +1,127 @@
+"""
+Assist integrated channels with retrieving course metadata.
+
+Module contains resources for integrated pipelines to retrieve all the
+metadata for courses in the course catalog belonging to a particular
+enterprise customer.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import json
+from logging import getLogger
+
+from django.contrib.auth.models import User
+from slumber.exceptions import SlumberHttpBaseException
+
+from enterprise.course_catalog_api import CourseCatalogApiClient
+from enterprise.models import EnterpriseCustomer
+
+
+EXCLUDED_COURSE_DETAIL_KEYS = [
+    'course_runs',
+]
+
+
+LOGGER = getLogger(__name__)
+
+
+def updated_course_run(course_details, course_run_details):
```

Comment:
This function is named like a property, but it's performing an action -- maybe better to name it `update_course_run_details`?